### DISCUSSION TITLE
Why does the penguin emoji look like a puffin? 🐧

### DISCUSSION THREAD

**Discussion by NashJames** [community/community#42517] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I honestly thought for a while the lack of a penguin emoji was a deliberate choice. It seems like the design is a mashup of various penguin breeds, and accidentally looks like something else.

The red beak with the sharp downturn at the end is pretty recognisable to a puffin. I know zoomed in the beak is a dark orange, but it definitely merges into red when viewed normally.

Apologies for the criticism. Appreciate your work.

**Comment by jfullstackdev** [comment, marked as answer]
you can collaborate in here 

https://github.com/ikatyang/emoji-cheat-sheet

it's not actually purely from GitHub, as you can see there
